polaris-react/src/components/Badge/components/Pip/Pip.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import {classNames, variationName} from '../../../../utilities/css';
+import type {Progress, Status} from '../../types';
+import {VisuallyHidden} from '../../../VisuallyHidden';
+import {useI18n} from '../../../../utilities/i18n';
+import {getDefaultAccessibilityLabel} from '../../utils';
+
+import styles from './Pip.scss';
+
+export interface PipProps {
+  status?: Status;
+  progress?: Progress;
+  accessibilityLabel?: string;
```

Comment:
I think `statusAndProgressLabelOverride` was a superb name 🙈

Reply:
Can we settle for `accessibilityLabelOverride`? :D

Reply:
You've got a deal